server/athenian/api/internal/team.py

```diff
@@ -223,3 +229,30 @@ async def delete_team(team: Row, sdb_conn: Connection) -> None:
     await sdb_conn.execute(sa.delete(Team).where(Team.id == team_id))
     # TeamGoal-s have been deleted by ON DELETE CASCADE on team_id, now empty Goals must be removed
     await delete_empty_goals(team[Team.owner_id.name], sdb_conn)
+
+
+async def sync_team_members(
+    team: Row,
+    members: Sequence[int],
+    sdb_conn: Connection,
+) -> Sequence[int]:
+    """Update the members of the Team if `members` contain new members.
+
+    Existing members are never removed.
+    Return the members added to the Team.
+
+    """
+    assert await conn_in_transaction(sdb_conn)
+    members_set = set(members)
+    db_members = team[Team.members.name]
+    new_members = members_set.difference(db_members)
+
+    if new_members:
+        all_members = sorted(chain(new_members, db_members))
+        await sdb_conn.execute(
+            sa.update(Team)
+            .where(Team.id == team[Team.id.name])
+            .values({Team.updated_at: datetime.now(timezone.utc), Team.members: all_members}),
+        )
```

Comment:
We should rewrite this to write the new team members without any special logic inside.

Who are the new team members is the caller's responsibility.

Reply:
What if I call the function `add_team_members` ? 
If needed I can split another _set_team_members without logic inside